lobby/src/main/java/org/triplea/lobby/server/controller/rest/ControllerHub.java

```diff
@@ -0,0 +1,17 @@
+package org.triplea.lobby.server.controller.rest;
+
+import static spark.Spark.path;
+
+import org.triplea.lobby.server.config.LobbyConfiguration;
+import org.triplea.lobby.server.db.Database;
+
+/**
+ * Main Routing class to connect all Controllers together.
+ */
+public class ControllerHub {
+
+  public static void initializeControllers(final LobbyConfiguration configuration) {
+    final Database database = new Database(configuration);
+    path("/api/v0", new ModeratorActionController(database)::initializeRoutes);
```

Comment:
Noting the path, let's avoid version numbers in URLs please. URL names are some of the longest lasting entities in a system, attaching a version number to it can be very bad. We also do not need version numbers in URLs to do routing. My favorite strat is to do application load balancing on versions and have all versions go to specific servers. This allows for code to be updated without almost any backward compatibility concerns. A 'v0' in URL locks us in to supporting that for quite a lot longer and can create a painful migration path.